### DISCUSSION TITLE
Effect of grid resoolution on polarization uncertainties

### DISCUSSION THREAD

**Discussion by redeboer** [ComPWA/polarimetry#231] [General]
As reported in https://github.com/ComPWA/polarimetry/pull/229#issuecomment-1286149896, the uncertainties on polarization&nbsp;$\vec{P}$ seem to decrease when the $\vec\alpha$-field is computed from a grid sample instead of the full amplitude model. Here is an attempt to see whether an increase of the grid resolution brings the uncertainties closer together to the uncertainties determined by the full amplitude model and by the LHCb paper.

| Resolution | $\vec{P}$ | Source | JSON | TAR |
|:---:| --- |:---:| ---:| ---:|
| **100x100** | ![100](https://user-images.githubusercontent.com/29308176/197248452-5160f109-032d-478b-911a-7ee3e119da41.svg) | [[download]](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25323794/artifacts/browse)         |  68&nbsp;MB |  26&nbsp;MB
| **200x200** | ![200](https://user-images.githubusercontent.com/29308176/197248458-33717ce9-d5d5-4faa-a0e8-3705212a7081.svg) | [[download]](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25324698/artifacts/browse) | 272&nbsp;MB | 104&nbsp;MB
| **300x300** | ![300](https://user-images.githubusercontent.com/29308176/197248461-eec005ca-e62f-41aa-94f8-1e1ea0d7b2eb.svg) | [[download]](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25325661/artifacts/browse) | 613&nbsp;MB | 232&nbsp;MB
| **[full]**  | ![full](https://user-images.githubusercontent.com/29308176/197248462-d883306f-a722-40a7-88db-09acd625eaa2.svg) | [[download]](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25327287/artifacts/browse)
| **[LHCb]**  | ![LHCb](https://user-images.githubusercontent.com/29308176/197185033-bd59eba0-dde7-4c13-bd11-e31a4c324cd6.svg) | [[paper]](https://arxiv.org/pdf/2208.03262.pdf#page=20)

**Comment by redeboer** [comment]
After #232, the effect of grid size on the polarization uncertainty has become negligible!

**Reply by mmikhasenko** [reply]
Beautiful! It seems that we might even reduce the number of bins 😆 
I do not suggest testing it since our justification of this binning is the width of L1520. 100x100 of 30Mb tar is completely fine.
